knowledge-base/implementation-documentation/0010-release-version-1-vdms.md

```diff
@@ -0,0 +1,18 @@
+# Release VDMs for 1.0
+
+In general, this should be avoided, because we use caret version `^1.54.0`.
+Even if we publish `1.55.0` sdk core, releasing clients are not necessary.
+
+## When releasing 1.0 VDMs should happen
+
+Only if when our generated code contains vulnerable code, we need to fix the generator and release new clients.
+This should not happen, and should be treated as edge case.
+When necessary, please avoid the following issue:
+
+- when publishing new VDMs for 1.0, the `latest` tag should still point to version 2. Maybe talking to the assembly team for additional information as they handle the bom PR. The worst case would be we release version 1 VDMs followed by version 2 VDMs, as a work around.
+
+## How to plan the support of releasing 1.0 VDMs
```

Comment:
I dont understand this heading. Is this meant as a preemptive thing or a thing that needs to be done when the edge case happens?

Reply:
Edge cases cannot be 100% avoided, in general.
In the previous meeting, there was a concern about the potential version 1 VDM release.
As a developer, the decision might not be under our control.
Therefore, I added a section about some processes and dependencies.

Reply:
Sorry, but I still don't get if this section should be a "In the unlikely case of emergency, follow these steps" or rather "Preparations that need to be done now so we are prepared if something happens".

Reply:
Discussed with @marikaner , let's keep this internally.